cmd/cli.go

```diff
@@ -18,8 +18,7 @@ var flags struct {
 }
 
 func printVersionAndLicense(file io.Writer) {
-	fmt.Fprintf(file, "cpackget version %v\n", version)
-	fmt.Fprintf(file, "%v\n", license)
+	fmt.Fprintf(file, "cpackget version %v %s\n", version, copyRight)
```

Comment:
the version variable contains a leading "v" unnecessarily, I think.

Reply:
Ack, makes sense not to have it duplicated. I've pushed a new update to this patch. The "v" from the version variable is removed prior to printing it out.